frontend/src/features/admin-form/settings/SettingsPage.tsx

```diff
@@ -59,6 +69,7 @@ export const SettingsPage = (): JSX.Element => {
         variant="line"
         py={{ base: '2.5rem', lg: '3.125rem' }}
         px={{ base: '1.5rem', md: '1.75rem', lg: '2rem' }}
+        defaultIndex={tabIndex === -1 ? 0 : tabIndex}
```

Comment:
should we also account for the case where the displayPayments is false?

Reply:
I agree with @wanlingt here! Currently if displayPayments is false we will show a blank settings content, but should be better to just redirect back to general (consistent with params of non-existent tab)

Reply:
thanks both, true! and will fix haha

Reply:
done in [19e2b8b](https://github.com/opengovsg/FormSG/pull/6348/commits/19e2b8bea00399ede3ea3dbf9874a751ccbff0f5)!